internal/service/advancedcluster/testdata/TestAccMockableAdvancedCluster_shardedAddAnalyticsAndAutoScaling/02_01_PATCH__api_atlas_v2_groups_{groupId}_clusters_{clusterName}_2024-10-23.json

```diff
@@ -5,6 +5,7 @@
     {
      "analyticsSpecs": {
       "diskIOPS": 2000,
+      "diskSizeGB": 40,
```

Comment:
`disk_size_gb` was still preserved in the known after apply logic (together with root level attributes handling), this is why it is now present in the PATCH request even though not defined in the terraform config. Don't see risk in this case.